From 99f7a588ee827e25a153ac066a4f80f13a86e10e Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anuj.mittal@intel.com>
Date: Mon, 29 Jul 2019 12:27:19 +0800
Subject: [PATCH] modules/vulkan: point to correct location of spv shaders

They are not generated and are present in source directory.

Upstream-Status: Submitted [https://github.com/intel/libxcam/pull/654]

Signed-off-by: Anuj Mittal <anuj.mittal@intel.com>
---
 modules/vulkan/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/vulkan/Makefile.am b/modules/vulkan/Makefile.am
index 22763f0..56578ec 100644
--- a/modules/vulkan/Makefile.am
+++ b/modules/vulkan/Makefile.am
@@ -5,7 +5,7 @@ XCAM_VK_CXXFLAGS = \
     $(LIBVULKAN_CFLAGS)           \
     -I$(top_srcdir)/xcore         \
     -I$(top_srcdir)/modules       \
-    -I$(top_builddir)/shaders/spv \
+    -I$(top_srcdir)/shaders/spv \
     $(NULL)
 
 XCAM_VK_LIBS = \
-- 
2.7.4

